infrastructure/template.yaml

```diff
@@ -1056,60 +1065,225 @@ Resources:
     Properties:
       EventBusName: !Ref CheckHmrcEventBus
       Description: Capture the audit event RESPONSE_RECEIVED
-      State: DISABLED
+      State: !If [IsDevLikeEnvironment, ENABLED, DISABLED]
```

Comment:
What's the reason we only want these in dev?

Reply:
This was part of the acceptance criteria, I think this was due to the fact that the properties inside the current events are not yet final. A further talk with TXMA to get exact details are required.